Tests/feaLib/builder_test.py

```diff
@@ -51,7 +51,7 @@ def makeTTFont():
 
 class BuilderTest(unittest.TestCase):
     # Feature files in data/*.fea; output gets compared to data/*.ttx.
-    TEST_FEATURE_FILES = """
+    TEST_FEATURE_FILES = os.getenv("TEST_FEATURE_FILES","""
```

Comment:
You don't need to read from an environment variable to select a specific test to run. With pytest you can pass the `-k` option whtere the argument are test names (or parts of their name). E.g.

```
$ pytest -k MultipleLookupsPerGlyph2
```

Or you can be more specific and only select one test file and restrict to the test named such:

```
$ pytest Tests/feaLib/builder_test.py -k MultipleLookupsPerGlyph2
```

If you're using `tox` to run test suite, you can also pass parameters to pytest. Anything after `--` is passed on to pytest:

```
$ tox -e py37-cov -- Tests/feaLib/builder_test.py -k MultipleLookupsPerGlyph2
```

This is standard tox and pytest, documented in their respecitve documentation. I prefer this over custom use of environment vars.

Reply:
Oh great - I'll take this commit out.